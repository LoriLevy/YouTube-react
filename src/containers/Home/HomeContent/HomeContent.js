import React from "react";
import { VideoGrid } from "../../../components/VideoGrid/VideoGrid";
import "./HomeContent.scss";
import "./HomeContent-mq.scss";

export class HomeContent extends React.Component {
    render() {
      return (
        <div className="home-content">
            <div className="responsive-video-grid-container">
                <VideoGrid title="Trending" />
                <VideoGrid title="Antique Dolls" hideDivider={true} />
            </div>
        </div>
      );
    }
  }
