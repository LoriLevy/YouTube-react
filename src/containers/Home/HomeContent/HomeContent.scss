@import '../../../styles/shared.scss';

.home-content {
    margin-left: $sidebar-left-width;
    display: grid;
    grid: auto / auto;
    justify-content: center;
}

