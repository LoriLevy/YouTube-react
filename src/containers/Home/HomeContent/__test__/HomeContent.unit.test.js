import React from 'react';
import { shallow } from 'enzyme';
import { HomeContent } from '../HomeContent';

describe('HomeContent', () => {
    test('it renders', () => {
      const wrapper = shallow(<HomeContent />);
      expect(wrapper).toMatchSnapshot();
    });
  });