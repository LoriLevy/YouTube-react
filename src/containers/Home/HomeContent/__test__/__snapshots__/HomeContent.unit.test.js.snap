// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeContent it renders 1`] = `
<div
  className="home-content"
>
  <div
    className="responsive-video-grid-container"
  >
    <VideoGrid
      title="Trending"
    />
    <VideoGrid
      hideDivider={true}
      title="Antique Dolls"
    />
  </div>
</div>
`;
